Xamarin.Essentials/Browser/Browser.shared.cs

```diff
@@ -35,9 +60,56 @@ internal static Uri EscapeUri(Uri uri)
         }
     }
 
+    public class BrowserLaunchOptions : IEquatable<BrowserLaunchOptions>
```

Comment:
Any reason this is IEquatable? Doesn't seem like it needs to be at all since it is a class.... should it be a struct?

Reply:
Probably shod be class with no interfaces. Not sure if this is official, but I go with POCO classes in, read-only structs out...

Reply:
See my other comment. While all current features use the style @mattleibow describes I think it would be wortwhile investigating when we should differ from this. Even though this is an async method it could be that a passed value type does not have to be boxed. Then, having this as a value type would IMHO be preferrable due to the lessened heap allocation. All contained properties are value types as well, which is not the case for any other in parameter for the public api.